### ISSUE TITLE
Looping playback not reliable

### ISSUE THREAD

**Issue by alanlorence** [videovillage/Screen-Issues#45]
Sometimes looping playback works, sometimes it doesn't.  If no In/Out points are set, looping almost never works. Even with In/Out set, often times playback stops after a few loops.

**Comment by adamwilt** [comment]
Agreed: Looping frequently doesn't work. Set Looping on; play the clip, it stops at the end. It may take several play attempts before looping starts. Once looping starts, it may stop after four loop, or nine, or a few dozen, but it almost never continues. 

This happens with no in/out points, with just an in or an out point, and with both set. When clips loop, the play/pause button briefly changes from pause to play when the playhead resets to the start of the clip. Usually when looping fails, the playhead resets to the start (or inpoint) and the button changes to play, but sometimes (rarely) the loop will stop at/near the outpoint, and the button is still pause; it needs to be clicked twice (or the spacebar hit twice)before playing resumes.

Ping-ponging seems more reliable, but it also stops arbitrarily after a variable number of runs.  When ping-pong fails, it's at either end of the play range, and the play/pause button is always pause, and must be clicked twice to resume playback. (Ping-ponging also randomly overshoots either the inpoint or the outpoint, but one bug at a time!)

I see this behavior with ProRes, H.264, and H.265 clips, all in .MOV wrappers.

**Comment by gregcotten** [comment]
Taking a hard look at this right now!
